docs/plugins/vertex-ai.md

```diff
@@ -147,3 +156,36 @@ const llmResponse = await generate({
   prompt: 'What should I do when I visit Melbourne?',
 });
 ```
+
+### Evaluators
+
+To use the evaluators from Vertex AI Rapid Evaluation, add an `evaluation` block to your `vertexAI` plugin configuration.
+
+```js
+import { vertexAI, VertexAIEvaluationMetricType } from '@genkit-ai/vertexai';
+
+export default configureGenkit({
+  plugins: [
+    vertexAI({
+      projectId: 'your-cloud-project',
+      location: 'us-central1',
+      evaluation: {
+        metrics: [
+          VertexAIEvaluationMetricType.SAFETY,
+          {
+            type: VertexAIEvaluationMetricType.ROUGE,
+            metricSpec: {
+              rougeType: 'rougeLsum',
+            },
+          },
+        ],
+      },
+    }),
+  ],
+  // ...
+});
+```
+
+The configuration above adds evaluators for the `Safety` and `ROUGE` metrics. The example shows two approaches- the `Safety` metric uses the default specification, whereas the `ROUGE` metric provides a customized specification that sets the rouge type to `rougeLsum`.
+
+Both evaluators can be run using the `genkit eval:run` with a compatible dataset i.e. a datset with `output` and `reference` fields. The `Safety` evaluator can also be run using the `genkit eval:flow -e vertexai/safety` command since it only requires an `output`.
```

Comment:
```suggestion
Both evaluators can be run using the `genkit eval:run` command with a compatible dataset: that is, a dataset with `output` and `reference` fields. The `Safety` evaluator can also be run using the `genkit eval:flow -e vertexai/safety` command since it only requires an `output`.
```